src/inc/stackstring.h

```diff
@@ -0,0 +1,196 @@
+// Licensed to the .NET Foundation under one or more agreements. 
+// The .NET Foundation licenses this file to you under the MIT license. 
+// See the LICENSE file in the project root for more information. 
+
+
+#ifndef __STACKSTRING_H_
```

Comment:
This file is pretty much the duplicate of what lives under pal\src\include\ folder. The only difference I can see is that a call to PAL_Free has been changed to free and ExtractString function implemented.

I think it will be better to have a single source implementation that can be shared between PAL and VM. Any future bug/fixes/updates will automatically get reflected for both components and not just fine.

CC @janvorli